test/test_helper.rb

```diff
@@ -97,3 +97,17 @@ def body_for(topic)
 
   parts[2]
 end
+
+def assert_oxford_comma(text)
```

Comment:
Love this assertion! Since it isn't used outside `test/topics_test.rb`, could you define it there?

Reply:
Hmm, all the helpers here are only used in topics_test, as it's the only test file we have. 😀  For consistency, and to keep the tests separate from the things that support the tests, I'd like to keep this here.